Criterion/Report.hs

```diff
@@ -120,8 +137,10 @@ formatReport reports templateName = do
     templates <- getTemplateDir
     template <- includeTemplate (includeFile [templates]) template0
 
+
+    let reportJSON = TL.toStrict . TL.concatMap escapeJSON . encodeToLazyText
     let context = object
-            [ "json"                .= reports
+            [ "json"                .= String (reportJSON reports)
```

Comment:
Is the use of `String` here needed? The `ToJSON` instance for `Text` should take care of this, if I recall correctly.

Reply:
It might not be, I started out with `ByteString` and got type errors. I’ll see if it works without the wrapper.